packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleTextViewer/SingleTextViewer.js

```diff
@@ -1,6 +1,6 @@
 import { Box } from 'grommet'
 import PropTypes from 'prop-types'
-import styled from 'styled-components'
+import styled, { withTheme } from 'styled-components'
```

Comment:
```suggestion
import styled, { useTheme } from 'styled-components'
```

I'd prefer a hook here, rather than a HOC.